src/assets/icons/16/icon-apps-primary.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Je pense que c'est problématique d'avoir les SVG avec de la couleur on devrait pouvoir avoir la même SVG pour les 2 themes. Mais c'est à voir dans de l'optimisation dans un futur plus ou moins proche...

Reply:
Did you try to use the `fill` css property? Or maybe a SVGr component like for [Home icon](https://github.com/cozy/cozy-bar/blob/master/src/components/Apps/IconCozyHome.jsx)?

Reply:
It's more complex because we used it on (background-image](https://github.com/cozy/cozy-bar/pull/451/files#diff-2cb59efd08fdd9e7511322df3541313eR21). 

With my comment I wanted to warn you but not necessarily solve the problem in this PR

Reply:
Maybe we can avoid using it as a background-image? Did you try it?
This issue should be solved in this PR instead of keeping it in the code for a while IMHO

Reply:
@CPatchane I used an `Icon` from cozy-ui instead of the background-image svg. Is it OK for you like this?